package.json

```diff
@@ -19,12 +19,15 @@
     "clean": "node scripts/clean-modules/index.js",
     "clean:install": "npm run clean && npm install",
     "build:test": "NODE_ENV=test webpack --config webpack.config.js",
-    "build:production": "NODE_ENV=production webpack --config webpack.config.js -p",
+    "build:production": "NODE_ENV=production node --max_old_space_size=2000 node_modules/webpack/bin/webpack.js --config webpack.prod.config.js -p",
```

Comment:
webpack was running out of memory optimizing for prod. I'd like to do more investigation here, but for the time being i bumped up the heap size for the process.